modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -470,6 +433,18 @@ public ObjectDefinition putObjectDefinition(
 				titleServiceBuilderObjectField.getObjectFieldId();
 		}
 
+		Status status = objectDefinition.getStatus();
+
+		if (status == null) {
+			int statusInt = serviceBuilderObjectDefinition.getStatus();
+
+			status = new Status() {
+				{
+					code = statusInt;
+				}
+			};
+		}
+
```

Comment:
I didn't want to build a new object just to get the code after.

I trying to think in a different way, but couldn't name a variable :S Here's the sniped

```
int status = serviceBuilderObjectDefinition.getStatus();

if (objectDefinition.getStatus() != null) {

	Status dontKnowTheNameHere = serviceBuilderObjectDefinition.getStatus();

	status = dontKnowTheNameHere.getCode();
	
}
```

Reply:
Agreed, done here: https://github.com/liferay-objects/liferay-portal/pull/3054/commits/3af8cf956e1e80b803b18292e907789fd905104b

I followed the same name pattern used [here](https://github.com/liferay/liferay-portal/blob/6ce0b9eb049e99a50285a324a3bd4afd76d340ab/modules/apps/static/portal-osgi-web/portal-osgi-web-portlet-container-test/src/testIntegration/java/com/liferay/portal/osgi/web/portlet/container/test/PublicRenderParameterTest.java#L124), when there is a name conflict between an object and a primitive variable we should add the suffix on the primitive variable where the suffix is the type.